extensions/wueadress/src/index.tsx

```diff
@@ -0,0 +1,160 @@
+import { ActionPanel, Action, List } from "@raycast/api";
+import { useState, useEffect } from "react";
+import axios from "axios";
+import * as cheerio from "cheerio";
+import { wueIconData } from "./imageData";
+
+interface SearchResult {
+  title: string;
+  phones: string[];
+  emails: string[];
+  workplaces: string[];
+  academicTitle: string;
+}
+
+export default function Command() {
+  const [searchQuery, setSearchQuery] = useState<string>("");
+  const [searchResults, setSearchResults] = useState<SearchResult[] | undefined>(undefined);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [error, setError] = useState<string | undefined>(undefined);
+
+  const fetchHTMLContent = async (query: string) => {
+    setIsLoading(true);
+    setError(undefined);
+
+    try {
+      const url = "https://wueaddress.uni-wuerzburg.de/search/person";
+      const params = query ? { q: query } : undefined;
+      const response = await axios.get(url, { params });
+
+      const results = collectData(response.data);
+      setSearchResults(results);
+      setError(undefined);
+    } catch (error) {
+      setSearchResults(undefined);
+      setError((error as Error).message);
+    } finally {
+      setIsLoading(false);
+    }
+  };
+
+  useEffect(() => {
+    if (searchQuery) {
+      fetchHTMLContent(searchQuery);
+    } else {
+      setSearchResults(undefined);
+      setError(undefined);
+    }
+  }, [searchQuery]);
+
+  return (
+    <List
+      isLoading={isLoading}
+      onSearchTextChange={setSearchQuery}
+      searchBarPlaceholder="Search for a contact..."
+      navigationTitle="WueAddress"
```

Comment:
```suggestion
```
Don't change the navigationTitle in the root command - it will be automatically set to the command name. Use navigationTitle only in nested screens to provide additional context.

Reply:
Made changes in commit